specification/storage/resource-manager/Microsoft.Storage/stable/2021-01-01/storage.json

```diff
@@ -2208,11 +2259,11 @@
         "state": {
           "type": "string",
           "enum": [
-            "provisioning",
-            "deprovisioning",
-            "succeeded",
-            "failed",
-            "networkSourceDeleted"
+            "Provisioning",
+            "Deprovisioning",
+            "Succeeded",
+            "Failed",
+            "NetworkSourceDeleted"
```

Comment:
This is breaking, anyway should be fine for new API version.
Better to make "modelAsString":  from false to true, to avoid breaking in the future.

Reply:
Done.